examples/privatekeys/signTransaction.go

```diff
@@ -0,0 +1,46 @@
+// Package main demonstrates an API client which returns the UserID of its API key.
+package main
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/tkhq/go-sdk"
+	"github.com/tkhq/go-sdk/pkg/api/client/private_keys"
+
+	"github.com/tkhq/go-sdk/pkg/api/models"
+)
+
+func main() {
+	// NB: make sure to create and register an API key, first.
+	client, err := sdk.New("default")
+	if err != nil {
+		log.Fatal("failed to create new SDK client:", err)
+	}
+
+	timestamp := time.Now().UnixMilli()
+	timestampString := strconv.FormatInt(timestamp, 10)
+
+	pkId := "<PRIVATE KEY ID>"
```

Comment:
nit: I'd prefer to change the variable name to be descriptive, rather than setting its value to a bogus description... something like:
```go
var privateKeyID string
var unsignedTransaction string
```